UnityProject/Assets/Scripts/Lighting/LightMountStates.cs

```diff
@@ -241,19 +249,25 @@ private void OnDamageReceived(DamageInfo arg0)
 
 	private void SyncLightState(LightMountState oldState, LightMountState newState)
 	{
-		state = newState;
+		//state = newState;
 		ChangeLightState(newState);
 	}
 
 	public override void OnStartClient()
 	{
-		SyncLightState(state, state);
+		// Commenting this line out for now as you shouldn't be running a syncvar hook outside of the syncvar getting updated data from server.
+		//SyncLightState(state, state);
 		base.OnStartClient();
 	}
 
 	[Server]
 	private void ServerChangeLightState(LightMountState newState)
 	{
+		if (!isServer) 
```

Comment:
The [Server] Attribute will stop this method being called on Client so !isServer condition is not needed. If it is called on the client though a warning is logged in console.

Reply:
Sorry about that one. Again shotgunned the approach and put one there. To be honest I thought all the [Server] attribute did was give a warning when the client attempted access but apparently it also prevents it.